vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_list_updater.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_PUBLISHER_PUBLISHER_LIST_UPDATER_H_
+#define BRAVELEDGER_PUBLISHER_PUBLISHER_LIST_UPDATER_H_
+
+#include <functional>
+#include <map>
+#include <string>
+
+#include "base/timer/timer.h"
+#include "bat/ledger/ledger.h"
+
+namespace bat_ledger {
+class LedgerImpl;
+}
+
+namespace braveledger_publisher {
+
+// Automatically updates the publisher prefix list store on regular
+// intervals.
+class PublisherListUpdater {
+ public:
+  explicit PublisherListUpdater(bat_ledger::LedgerImpl* ledger);
+
+  PublisherListUpdater(const PublisherListUpdater&) = delete;
+  PublisherListUpdater& operator=(const PublisherListUpdater&) = delete;
+
+  ~PublisherListUpdater();
+
+  // Starts the auto updater
+  void StartAutoUpdate();
+
+  // Cancels the auto updater
+  void StopAutoUpdate();
+
+  using OnPublisherListUpdatedCallback = std::function<void()>;
+
+  // Sets a callback that will be executed after the publisher
+  // list has been updated
+  void SetOnPublisherListUpdatedCallback(
+      OnPublisherListUpdatedCallback callback);
+
+ private:
+  void StartFetchTimer(
+      const base::Location& posted_from,
+      base::TimeDelta delay);
```

Comment:
`const base::TimeDelta&`

Reply:
According to the [code comments](https://cs.chromium.org/chromium/src/base/time/time.h?q=timedelta&sq=package:chromium&g=0&l=24) in `time.h`, we should prefer passing `TimeDelta` by value.